tket/src/Utils/include/Utils/Assert.hpp

```diff
@@ -33,19 +33,22 @@
  * to a stringstream) will NOT begin if `condition` is true,
  * so there is no performance penalty.
  *
- * Note: the intention was that the code would be ignored by test code
- * coverage, even if multiline. However that didn't work, so we may
- * just manually surround the worst multiline offenders until we come up
- * with a better solution.
- *
  * This also checks if evaluating `condition` itself throws an exception.
  *
- * Note: we already tried having another version which threw exceptions
- * instead of aborting. However, it led to problems:
+ * Notes: (1) single line code is ignored by test code coverage.
+ * However, multiline code is not. Currently we just manually surround the
+ * worst multiline offenders with start/stop tags.
+ * (2) We tried putting the start/stop tags within the macro, to make test
+ * coverage ignore the code; unfortunately this did NOT work.
+ * We suspect that it's because comments are stripped from the macro before
+ * test coverage sees the code, but we don't know. See TKET-1856.
```

Comment:
Remove reference to internal Jira task please.

Reply:
We use https://github.com/CQCL/tket/issues ! Opening an issue there automatically creates a mirror issue on our internal Jira.